### PR TITLE
add mbstrdecoder

### PR DESCRIPTION
**Pull request by basnijholt**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

Depends on releasecmd https://github.com/conda-forge/staged-recipes/pull/12327.

### CODE REVIEW

**Comment by basnijholt** [comment]
Depends on https://github.com/conda-forge/staged-recipes/pull/12327.

File: mbstrdecoder/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+    - releasecmd
```
Comment (dopplershift) [inline mbstrdecoder/meta.yaml:22]: Is this really needed? The recipe was able to pass without this being available in conda-forge...

File: mbstrdecoder/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+{% set name = "mbstrdecoder" %}
```
Comment (synapticarbors) [inline mbstrdecoder/meta.yaml:1]: file in incorrect location. Needs to be located in `recipes/mbstrdecoder/`

File: mbstrdecoder/meta.yaml (hunk 1 above)
Comment (basnijholt) [reply mbstrdecoder/meta.yaml]: The recipe was in the incorrect location... That's why it passed.

**Comment by synapticarbors** [comment]
closed by #12338

**Pull request closed without merging**